### ISSUE TITLE
Minor enhancements to the task editor UI

### ISSUE THREAD

**Issue by deifactor** [deifactor/wrtsc#20]
- [ ] Support for having 0x of a task
- [ ] Enable/disable tasks
- [ ] Some way to edit multiples at a time
- [ ] Directly edit the number of tasks

Saving/loading task lists goes in a separate PR since that's bigger.

**Comment by deifactor** [comment]
Depends on #24 since this is going to require modifying how saves are stored.
